ADMozaikCollectionViewLayout/Private/ADMozaikLayoutCache.swift

```diff
@@ -82,6 +82,9 @@ class ADMozaikLayoutCache {
         if let size = self.cachedSizeOfItemAtIndexPathDictionary[indexPath] {
             return size
         }
-        return self.mozaikLayoutDelegate.collectionView(self.collectionView, mozaik: layout, mozaikSizeForItemAt: indexPath)
+        guard let delegate = self.mozaikLayoutDelegate else {
```

Comment:
I think it makes sense to check this before let above as this is a required parameter.